copier.yml

```diff
@@ -62,8 +62,29 @@ component_owner:
     type: str
     help: |
         Owner of the component in the Developer Portal Software Catalogue.
-        This will normally be group:default/<your_group>
-    default: "group:default/sscc"
+        This will normally be group:default/<your group>
+
+component_type:
+    type: str
+    help: |
+        Type of the component in the Developer Portal Software Catalogue.
+        Most likely service, library or application (without quotes).
```

Comment:
I will likely turn this one into a `choices` too, assuming they are unlikely to change often...?

Reply:
These are used to select on when rendering backstage UIs, so it's important they use a recognised value as explained in [the reference docs](https://dev-portal.diamond.ac.uk/guide/developer-portal-catalog/docs/references/component-spec/#type-required)

Reply:
For both this and the common lifecycle stages, it would be good to do a cleanup of the transgressors currently in the dev portal. Might be something to add to @GDYendell's review meetings.